Changelog.md

```diff
@@ -1,5 +1,65 @@
 # Changelog
 
+## 20230831 - Exports API
+
+**New models**
+- ExportsCreateRequestBody
+- ExportBase
+- Export
+- ExportsCreateResponseBody
+- ExportVoucher
```

Comment:
Did we have to extract from`ExportVoucher` type`ExportVoucherParameters`? What's the purpose of that? You did not do that in SDK. Now we have more small types and need to jump around them to figure out the final payload. The same for other types here...

![image](https://github.com/voucherifyio/voucherify-openapi/assets/35666945/ea50ab82-1640-4670-908b-58573a375cc4)